idb/postgres/internal/writer/writer.go

```diff
@@ -232,28 +245,30 @@ func getTransactionParticipants(stxnad transactions.SignedTxnWithAD) (res []basi
 			}
 			res = append(res, address)
 		}
-	} else {
-		// inner transactions might have inner transactions might have inner...
-		// so the resultant slice is created after collecting all the data from nested transactions.
-		// this is probably a bit slower than the default case due to two mem allocs and additional iterations
-		size := acctsPerTxn * (1 + len(stxnad.ApplyData.EvalDelta.InnerTxns)) // approx
-		participants := make(map[basics.Address]struct{}, size)
-		add = func(address basics.Address) {
-			if address.IsZero() {
-				return
-			}
-			participants[address] = struct{}{}
+
```

Comment:
Thanks. Could you also delete `var add func(address basics.Address)` above and not use the named return value? There is no reason.